exe-common/src/network/hermes.rs

```diff
@@ -26,29 +26,45 @@ impl HermesEndPoint {
     }
 
     pub fn uri(& mut self, path: &str) -> String {
-        format!("{}/{}/{}", self.url, self.blockchain, path)
+        format!("{}/{}", self.url, path)
     }
 }
 
 impl Api for HermesEndPoint {
     fn get_tip(&mut self) -> Result<BlockHeader> {
-        let uri = self.uri("tip").as_str().parse().unwrap();
+        let uri = self.uri("tip");
         info!("querying uri: {}", uri);
 
+        let mut err = None;
+
+        let now = SystemTime::now();
+
         let mut bh_bytes = Vec::with_capacity(4096);
         {
             let client = Client::new(&self.core.handle());
-            let work = client.get(uri).and_then(|res| {
-                res.body().for_each(|chunk| {
+            let work = client.get(uri.parse().unwrap()).from_err::<Error>()
+                .and_then(|res| {
+                println!("Response: {}", res.status());
```

Comment:
no `println` please, you `debug!` or `info!` or `warn!` or `error!`

Reply:
Oops, I forgot to remove that line! Thanks.